sdk/cognitiveservices/cognitiveservices-luis-authoring/test/helper.ts

```diff
@@ -0,0 +1,41 @@
+import * as msRestNodeAuth from "@azure/ms-rest-nodeauth";
+import { LUISAuthoringClient } from "../src/lUISAuthoringClient";
+
+var global_client: LUISAuthoringClient;
+var subscription = "b15ebe3a1ec446a08f8021fe6f95f0f6";
+
+export function interactiveLogin() {
+    return msRestNodeAuth.interactiveLogin().then((credentials: any) => {
```

Comment:
How are we expecting the interactive login to work in CI?

Reply:
we should change this in a future PR. the purpose of this PR is to make sure the provided structure (using mocha and typescript) fits the SDK testing in this repo.